src/content/releases/_index.en.md

```diff
@@ -5,6 +5,20 @@ pre = "<b>5. </b>"
 weight = 25
 +++
 
+## v0.6.0 Improved Submariner High Availability and various Lighthouse enhancements
+> This release mainly focused on support for headless Services and StatefulSets in Lighthouse,
+> as well as improving Submariner's High Availability (HA).
+
+* **Lighthouse** has been enhanced to:
+  * Be aware of the local cluster Gateway connectivity, and now does not announce
+    IP addresses for disconnected remote clusters.
+  * Support headless Services
+  * StatefulSets
```

Comment:
```suggestion
  * Be aware of pods' availability for a service, and now doesn't announce IP addresses for services that have zero active pods
```
StatefulSets not yet, will likely come in 0.6.1

Reply:
Why 0.6.1? we have released 0.6.0-rc0... we can wait to release 0.6.0, right?, lets talk on team meeting :)

Reply:
We decided on 0.7.0 for StatefulSets.